sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobContainerAsyncClient.java

```diff
@@ -1045,4 +1045,47 @@ private boolean validateNoETag(BlobRequestConditions modifiedRequestConditions)
         }
         return modifiedRequestConditions.getIfMatch() == null && modifiedRequestConditions.getIfNoneMatch() == null;
     }
+
+    /**
+     * Restores a previously created container.  The restored container
```

Comment:
We're restoring a deleted container right?

```suggestion
     * Restores a previously delete container. The restored container
```

Reply:
done. thank you!